StorageGrain/StorageGrain.cs

```diff
@@ -0,0 +1,32 @@
+﻿using CommunAxiom.Commons.Client.Contracts.Grains.Storage;
+using Newtonsoft.Json.Linq;
+using Orleans.Runtime;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace StorageGrain
+{
+    // HACK: 
+    // [StorageProvider()]
+    public class StorageGrain : IStorageGrain
+    {
+        private readonly IPersistentState<JObject> _storageState;
+
+        public StorageGrain([PersistentState("storageGrain")] IPersistentState<JObject> storageState)
+        {
+            _storageState = storageState;
+        }
+
+        public Task<List<JObject>> GetData()
```

Comment:
Since we're saving a single JObject per grain instance (See SaveData) this method should only return a single jobject too